Styles/main.css

```diff
@@ -1,61 +1,92 @@
 @CHARSET "UTF-8";
 
-body{color:#801515;
-    background:#FFAAAA;
+body {
+    color: #801515;
+    background: #FFAAAA;
     font-family: 'Montserrat', sans-serif;
-    text-decoration: none;}
+}
 
-Width-wrap { border-left:1px solid black;
-    border-right:1px solid black;
-    border-bottom:1px  solid black;
-    padding-right: 2em; 
-    padding-left:  2em;
+.width-wrap {
+    border-left: 1px solid black;
+    border-right: 1px solid black;
+    border-bottom: 1px solid black;
+    padding-right: 2em;
+    padding-left: 2em;
     margin-bottom: 3em;
-    max-width: 1024px;}
+    max-width: 1024px;
+}
 
-Bio {text-indent: 3em;}
+.bio {
+    text-indent: 3em;
+}
 
-head{ color:#AA3939;}
+head {
+    color: #AA3939;
+}
 
-h1{ color:#801515;
-     background-image:url(/CSD120/images/gradient.png);
-     background-repeat: repeat-x;
-     font-family: 'Domine', serif;
-     font-family: 'Montserrat', sans-serif;}
+h1 {
+    color: #801515;
+    background-image: url(../images/gradient.png);

```

Comment:
Use a relative URL instead of a root-relative URL.  What you have will only work if your project is hosted on a server where the web site is inside a folder named CSD120.  What I have changed will work no matter what folder your website is hosted under.